src/chainparams.cpp

```diff
@@ -162,11 +162,12 @@ class CMainParams : public CChainParams {
             ( 1286500, uint256S("0x0000000002b9240e8e1ce741102c0d873148665e2aa45e5122fabf478b5c2fc2"))
             ( 1301500, uint256S("0x0000000003bafe8331f9753bc374f9c27539d640a1f86cbd25477e5732d1ff8c"))
             ( 1318100, uint256S("0x0000000000ad43247472917f4385eb168ea56e43d36c10443cca624331d06720"))
-            ( 1328300, uint256S("0x00000000005cec07cded2d161eb1b1578b1590675674c96cf028db5813a751da")),
-            1680860787,     // * UNIX timestamp of last checkpoint block
-            34706567,       // * total number of transactions between genesis and last checkpoint
+            ( 1328300, uint256S("0x00000000005cec07cded2d161eb1b1578b1590675674c96cf028db5813a751da"))
```

Comment:
Since we want this 4.0.0-rc3 to add only one checkpoint to previous 3.3.1, the checkpoint with height 1328300 should be dropped (it was introudeced with 4.0.0-rc2).

Reply:
Makes total sense, applied your suggestion, thanks